tests/TestBundle.hs

```diff
@@ -51,9 +51,8 @@ assertBundles support inputFiles outputFile = do
   case result of
     Left errs -> expectationFailure . P.prettyPrintMultipleErrors P.defaultPPEOptions $ errs
     Right _ -> do
-      process <- findNodeProcess
-      jsFiles <- Glob.globDir1 (Glob.compile "**/*.js") modulesDir
-      let entryPoint = modulesDir </> "index.js"
+      jsFiles <- concat <$> Glob.globDir [Glob.compile "*/*.js", Glob.compile "*/foreign.cjs"] modulesDir
+      let entryPoint = modulesDir </> "index.cjs"
```

Comment:
Not sure if this matters since it's in the bundler, but I'll comment in case it does matter. Shouldn't this be `index.js`?

Reply:
IIUC, this imperfectly-named variable is the *output* of the bundler, which is a CommonJS module, and we seem to be sticking to the convention that `.js` means ES module and `.cjs` means CommonJS module.